guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc

```diff
@@ -0,0 +1,36 @@
+[id="variables-for-generation-of-conversion-data_{context}"]
+= Variables for Generation of Conversion Data
+
+Before you run the Ansible role to generate conversion data, configure values of the following required Ansible variables.
+
+{Project} imports most of the required Ansible variables from the `{ansible-namespace}.convert2rhel` role.
+However, some of the required variables are not imported.
+These variables are marked with an asterisk * in the table below.
+You must create those additional variables manually and assign them to the `{ansible-namespace}.convert2rhel` role.
+
+.Required variables for conversion
+[cols="20%,5%,20%",options="header"]
+|====
+| Name | Type | Intent and value
+| `{project-context}_server_url` * | string | URL of your {ProjectServer}
+| `{project-context}_username` * | string | Your admin login, such as `admin`
+| `{project-context}_password` * | string | Your admin password
+| `{project-context}_organization` * | string | Name of your organization
+| `{project-context}_rhel_wait_for_syncs` * | boolean | Must be set to `false`
+| `{project-context}_validate_certs` * | boolean | Set to `true` if you wish to enable certificate checks in Ansible. (default: `true`)
+| `{project-context}_convert2rhel_manage_subscription` | boolean | Set to `false` if you already have a manifest on your {ProjectServer}. If you decide to upload a new manifest from disk, the current manifest will be overwritten. (default: `true`)
+| `{project-context}_content_rhel_enable_rhel7` | boolean | Enables {RHEL} 7 repositories. Set to `true` if you intend to convert hosts to {RHEL} 7.
+| `{project-context}_convert2rhel_enable_centos7` * | boolean | Set to `true` if you want to prepare conversion data for CentOS 7. Otherwise must be set to `false`.
+| `{project-context}_convert2rhel_enable_oracle7` | boolean | Set to `true` if you want to prepare conversion data for Oracle 7. Otherwise must be set to `false`.
+| `{project-context}_content_rhel_enable_rhel8` | boolean | Enables {RHEL} 8 repositories. Set to `true` if you intend to convert hosts to {RHEL} 8.
+| `{project-context}_convert2rhel_enable_centos8` * | boolean | Set to `true` if you want to prepare conversion data for CentOS 8. Otherwise must be set to `false`.
+| `{project-context}_convert2rhel_enable_oracle8` | boolean | Set to `true` if you want to prepare conversion data for Oracle 8. Otherwise must be set to `false`.
+|====
+
+.Optional variables for conversion
+[cols="20%,5%,20%",options="header"]
+|====
+| Name | Type | Intent and value
+| `{project-context}_manifest_path` * | string | Path to a manifest to upload from disk, such as `~/manifest.zip`. The path must be set if you upload a new manifest from disk with `{project-context}_convert2rhel_manage_subscription` set to `true`.
+| `{project-context}_content_rhel_rhel8_releasever` * | string | Minor release version, such as `8.4`. (default: latest)
```

Comment:
When I was trying to the conversion, I had problem with RHEL version,
doing conversion with `latest` repos didn't work, I had to enable `8.5` repos, then it worked.
See https://access.redhat.com/discussions/5676681

So maybe we should encourage users to set `_content_rhel_rhel8_releasever` by default to `8.5`, or atleast mention the issue here.